functions/src/utils.ts

```diff
@@ -36,8 +36,9 @@ export const writeUpdatesAsync = async (
   }
 }
 
-export const isProd =
-  admin.instanceId().app.options.projectId === 'mantic-markets'
+export const isProd = () => {
+  return admin.instanceId().app.options.projectId === 'mantic-markets'
+}
```

Comment:
I did this because, the way it was, if this file was imported and executed in a script before calling `initAdmin`, it would blow up.